docs/src/Module.doc.js

```diff
@@ -36,6 +36,23 @@ card(
         type: 'string',
         required: true,
       },
+      {
+        name: 'expandedIndex',
+        type: '?number',
+        required: false,
+        description: [
+          'The 0-based index of the item that the expand/collapse state can be controlled programatically from an external component',
```

Comment:
This wording is a little awkward, and we should mention how this needs to be updated. Consider something like "The 0-based index indicating the item that should currently be expanded. This must be updated via `onExpandedChange` to ensure the correct item is expanded."